src/features/deposit/components/DepositForm/index.tsx

```diff
@@ -375,11 +374,23 @@ export const DepositForm = ({ chainType }: { chainType?: ChainType }) => {
             userAddress,
             depositNearResult,
             depositEVMResult,
-            depositSolanaResult
+            depositSolanaResult,
+            depositTurboResult
+          )}
+        {userAddress &&
+          (network === BlockchainEnum.NEAR ||
+            network === BlockchainEnum.TURBOCHAIN) && (
+            <Deposits
+              chainName={
+                network === BlockchainEnum.NEAR
+                  ? "near"
+                  : network === BlockchainEnum.TURBOCHAIN
+                    ? "turbochain"
+                    : null
+              }
+              depositResult={depositNearResult ?? depositTurboResult}
+            />
```

Comment:
This kind of confusing. So it renders result of deposit operation. I assume this will be a part of that refactoring along with machines.

Putting that aside I think checking `userAddress` and `network` is not needed here, you just need to render a whatever result the machine returned.

Reply:
Yes, you are right.